### ISSUE TITLE
Podcast Promo a11y fixes

### ISSUE THREAD

**Issue by RichardPK** [BBC-archive/psammead#4108]
**Describe the bug**
- A11y acceptance criteria diverged between the Podcast Promos issue, and the Jira ticket / Zeplin designs
- Changes required are detailed below

Jira: https://jira.dev.bbc.co.uk/browse/WSMEDIA-295
Screen reader UX: https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5fa4500498f1b6435cb487b5

**Expected behaviour**
- [x] Section - should be aria-labelledby the same id that the H2 value has. (see screen reader UX)
- [x] H2 icon - element wrapping the svg, change it from i to a span. There is an i tag - this is for italics text etc, this isn’t semantic HTML, lets please swap this for a non semantic element if it’s needed https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i 
- [x] Text resize - Icons don’t scale. They should use REM values. 
- [x] Image has a figure element - this can be announced by some screen readers, while you can use a figure for just an image on it’s own its only usually used if there is a caption, it’s also not in the screen reader ux. Remove the figure el
- [x] In the screen reader UX the link just has the brand title as the announced content, we also have ‘episodes’ - this isn’t needed 
- [x] Is the link the correct font weight? And correct visited colour? 
- [x] Change bg colour to #fdfdfd
- [x] 'More episodes' is aria hidden it shouldn’t be, and the screen reader UX specifies a paragraph element for the content here, also lets remove the unsemantic HTML e.g. the i tag wrapping the SVG

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
